src/index.html

```diff
@@ -1,13 +1,54 @@
 <!doctype html>
 <html lang="en">
-<head>
-  <meta charset="UTF-8">
-  <meta name="viewport"
-        content="width=device-width, user-scalable=no, initial-scale=1.0, maximum-scale=1.0, minimum-scale=1.0">
-  <title>Product cards</title>
-  <link rel="stylesheet" href="./style.css">
-</head>
-<body>
-  <h1>Product cards</h1>
-</body>
+  <head>
+    <meta charset="UTF-8">
+    <meta name="viewport"
+          content="width=device-width, user-scalable=no, initial-scale=1.0, maximum-scale=1.0, minimum-scale=1.0">
+    <title>Product cards</title>
+    <link rel="stylesheet" href="./style.css">
+    <link rel="preconnect" href="https://fonts.googleapis.com">
+    <link rel="preconnect" href="https://fonts.gstatic.com">
+    <link href="https://fonts.googleapis.com/css2?family=Roboto:wght@500&display=swap" rel="stylesheet">
+  </head>
+  <body class="page">
+    <div class="page__card card" data-qa="card">
+      <div class="card__img">
+        <img
+          src="images/imac.jpeg"
+          width="160" height="134"
+          alt="Зображення монітору"
+        >
+      </div>
+
+      <h2 class="card__name">Моноблок APPLE A1419 iMac 27" Retina 5K (MNED2UA/A)</h2>
+
+      <div class="card__code">Код товара: 195434</div>
+
+      <div class="card__rating">
+        <div class="card__stars">
+          <div class="stars stars--4">
+            <div class="stars__star"></div>
+            <div class="stars__star"></div>
+            <div class="stars__star"></div>
+            <div class="stars__star"></div>
+            <div class="stars__star"></div>
+          </div>
+        </div>
+
+        <p class="card__reviews-amount">Отзывов: 5</p>
+      </div>
+
+      <div class="card__price">
+        <p class="card__text">Цена:</p>
+        <span class="card__total">69 999 грн</span>
```

Comment:
what is the difference here between spam and p?

Reply:
I thought it will be semantic more correctly to divide somehow this line in html code to text and price by tags